docs/tutorials/advanced/classification_of_strategies.rst

```diff
@@ -28,15 +28,30 @@ Note that instances of the class also have this classifier::
     >>> s.classifier == expected_dictionary
     True
 
-and that we can retrieve individual entries from that :code:`classifier` dictionary::
-
-    >>> s = axl.TitForTat
-    >>> s.classifier['memory_depth']
+The instance starts with a copy of the class's classifier dictionary, but is
+allowed to change this classifier dictionary at any point, and many
+strategies do so upon initialization.
+
+In addition to the classifier dictionary, each classifier is defined with
+some logic that maps classifier definitions to values.  To learn the
+classification of a strategy, we first look in the strategy's classifier
+dictionary, then if the key is not present, then we refer to this logic.
+This logic must be defined for a class, and not specific instances.
+
+To lookup the classifier of a strategy, using the classifier dict, or the
+strategy's logic as default, we use :code:`Classifiers().get(<classifier>,
+<strategy>)`::
+
+    >>> from axelrod.classifier import Classifiers
```

Comment:
Could we:

```
import axelrod.classifier as ac
```

and use `ac.Classifiers()`

Or possibly have `Classifiers` loaded in the `axelrod` namespace and:

```
axl.Classifiers()
```

Not entirely sure if either of the two are best solutions (so feel free to go with something else). I'd like to avoid importing in to the main namespace.

Reply:
I suggested somewhat similarly above that we could initialize `Classifiers()` once somewhere on library load with e.g. `Classifiers = _Classifiers()` and then we could simply use `axl.Classifiers` as needed.

Reply:
I'm following Marc's advice here, and loading into the axelrod namespace.

Reply:
Sounds good to me :+1: